addons/web/views/onboarding_templates.xml

```diff
@@ -0,0 +1,128 @@
+<odoo>
+    <data>
+        <template id="onboarding_container">
+            <div class="o_onboarding_container collapse in">
+                <t t-call-assets="web.assets_onboarding"/>
+                <div t-att-class="'o_onboarding ' + classes + (' collapse' if folded_by_default else ' collapse in')"
+                     t-att-style="'background-image:url(' + image + ')'">
+                    <div class="o_onboarding_wrap">
+                        <a t-if="not no_close_btn"
+                           data-toggle="collapse" href=".o_onboarding_container"
+                           class="pull-right o_onboarding_btn_close o_onboarding_hide_folded"
+                           type="action"
+                           t-att-data-model="close_model" t-att-data-method="close_method"
+                        >
+                            <i class="fa fa-times"/>
+                        </a>
+                        <div class="o_onboarding_steps">
+                            <span t-if="not no_fold_btn" class="o_onboarding_folded_title">
+                                Configuration steps:
+                            </span>
+                            <t t-raw="0"/>
+                        </div>
+                        <a t-if="not no_fold_btn or folded_by_default"
+                           data-toggle="collapse" href=".o_onboarding"
+                           type="action"
+                           t-att-data-model="fold_model" t-att-data-method="fold_method"
+                           class="o_onboarding_btn_fold text-right">
+                            <i class="fa fa-chevron-up"/>
+                        </a>
+                        <div class="o_onboarding_completed_message text-center">
+                            <i class="fa fa-check text-success mr8"/>
+                            <span class="h3 mr16">Nice work!</span>
+                            <div class="btn btn-primary btn-sm o_onboarding_btn_completed"
+                                 style="vertical-align: inherit;">Close
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </template>
+        <template id="onboarding_step">
+            <div t-att-class="'o_onboarding_step ' + ('o_onboarding_done' if done else 'o_onboarding_current')">
+                <t t-call="web.onboarding_confetti"/>
```

Comment:
What about conffeti in js ?

And we simplyfy the dom of this, so that we walsayw inline it.